Packages/com.unity.inputsystem/Documentation~/Gamepad.md

```diff
@@ -64,25 +60,76 @@ Increased frequency should lead to an increased number of events on the respecti
 
 ## Rumble
 
-    ////TODO: provide list of what platforms/devices we support rumble on
-
-Gamepads implement the `IDualMotorRumble` interface to control left and right motor speed. Note, however, that not all gamepads support rumble at this point (it is not supported on WebGL, for example).
+The [`Gamepad`](../api/UnityEngine.InputSystem.Gamepad.html) class implement the [`IDualMotorRumble`](../api/UnityEngine.InputSystem.Haptics.IDualMotorRumble.html) interface to allow you to control the left and right motor speeds.
 
 ```CSharp
 // Rumble the  low-frequency (left) motor at 1/4 speed and the high-frequency
 // (right) motor at 3/4 speed.
 Gamepad.current.SetMotorSpeeds(0.25f, 0.75f);
 ```
 
-## PS4
+>NOTE: Only the following combinations of devices/OSes currently support rumble:
+>* PS4, Xbox and Switch controllers when connected to their respective consoles.
+>* PS4 controllers when connected to mac or windows computers.
+>* Xbox controllers on windows.
+[//]: # (TODO: are we missing any supported configs?)
```

Comment:
Windows and UWP probably best mentioned separately (or as just `Windows/UWP`). ATM the two should match otherwise, though.

AFAIK we support rumble on Linux on all gamepads recognized by SDL.

Looking at Android and iOS, seems like we support rumble on neither ATM :(

Reply:
Added UWP, will ask Stefan to confirm what works on Linux.